chsdi/templates/htmlpopup/sis_planning.mako

```diff
@@ -16,9 +16,9 @@ coordinationlevel_text = 'coordlevel_text_%s' % lang
 planningstatus_text = 'plstatus_text_%s' % lang
 description_text = 'description_%s' % lang
 dateto = '-'
-datefrom = datetime.datetime.strptime(c['attributes']['validfrom'].strip(), "%Y-%m-%d").strftime("%d.%m.%Y")
+datefrom = datetime.datetime.strptime(c['attributes']['validfrom'].strip(), "%Y-%m-%dT%H:%M:%S").strftime("%d.%m.%Y")
```

Comment:
You could also  test if `validfrom` exists like you do for `validuntil` . Other wise there will be the same error fixed by @ltclm in https://github.com/geoadmin/mf-chsdi3/pull/2760/files